docker/cdt-infra-eclipse-full/ubuntu-18.04/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN mkdir /tmp/x \
     && find $JAVA11_HOME/include \
     && rm -rf /tmp/x
 
-# Get pre-built MacOSX toolchain from other image (this also 
+# Get pre-built MacOSX toolchain from other image (this also
```

Comment:
Auto formatting removed the trailing space, should I leave it like before?

Reply:
I'll build abs and test images - they take a long time if you have no build caches. For future you run build images script,and then deploy images then push. See readme in Docker dir.

Reply:
Oops, last comment was general. 

Leave the formatting fix.